iac/terraform/aws/auth/lambdas.tf

```diff
@@ -1,41 +1,21 @@
 locals {
+  lambda_archive_path = "${path.module}/../../../../core/services/aws/.build/zips"
   lambda_options = {
     refresh-token = {
-      name     = "refresh-token"
-      zip_path = "${path.module}/../../../../core/services/aws/.build/zips/refreshToken.zip"
-      policy   = data.aws_iam_policy_document.lambda_common_policy
-      handler  = "refreshToken.default"
+      name      = "refresh-token"
+      zip_path  = "${local.lambda_archive_path}/refreshToken.zip"
+      handler   = "refreshToken.default"
+      statement = local.policies.common_policies
       env_variables = {
```

Comment:
Run `make run-command-tf-fmt` to format terraform

Reply:
Updated in c7a8df7b774cad951c47e5d2a4ac76d8e394405c